crates/btcio/src/reader/tx_indexer.rs

```diff
@@ -156,6 +155,8 @@ mod test {
         }
     }
 
+    // Ignored because deposit request is not included as ops
+    #[ignore]
     #[test]
```

Comment:
```suggestion
    #[test]
    #[ignore = "Ignored because deposit request is not included as ops"]
```
Ref: https://doc.rust-lang.org/reference/attributes/testing.html#the-ignore-attribute